content/en/docs/tutorials/clusters/seccomp.md

```diff
@@ -35,16 +36,61 @@ profiles that give only the necessary privileges to your container processes.
 
 ## {{% heading "prerequisites" %}}
 
+{{< version-check >}}
+
 In order to complete all steps in this tutorial, you must install
 [kind](https://kind.sigs.k8s.io/docs/user/quick-start/) and
 [kubectl](/docs/tasks/tools/). This tutorial will show examples
-with both alpha (pre-v1.19) and generally available seccomp functionality, so
+with both alpha (pre-v1.22) and generally available seccomp functionality, so
```

Comment:
This phrase doesn't make sense. The alpha functionality demonstrated is _not_ pre-v1.22; it actually requires v1.22 to be able to follow every step.

```suggestion
both alpha (new in v1.22) and generally available seccomp functionality. You should
```